pkg/daemon/config_drift_monitor_test.go

```diff
@@ -0,0 +1,414 @@
+package daemon
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	ign3types "github.com/coreos/ignition/v2/config/v3_2/types"
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	"github.com/openshift/machine-config-operator/test/helpers"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/util/uuid"
+)
+
+func TestConfigDriftMonitor(t *testing.T) {
+	// Our errors. Their contents don't matter because we're not basing our
+	// assertions off of their contents. Instead, we're more concerned about
+	// their types.
+	fileErr := configDriftErr(fileConfigDriftErr(fmt.Errorf("file error")))
+	unitErr := configDriftErr(unitConfigDriftErr(fmt.Errorf("unit error")))
+
+	// Filesystem Mutators
+	// These don't have to be closures. I just wanted to avoid potential
```

Comment:
Can you re-frame the comment to not include "I" in sentence.

Reply:
Yes. In the future, I'll put "I" comments in the PR as a note to reviewers.